front/src/components/Loader.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import styles from './Loader.module.scss';
+import { useSelector } from 'react-redux';
+import { RootStateType } from '../redux/store';
+
+const Loader = (): JSX.Element | null => {
+  const { isFetching } = useSelector((state: RootStateType) => state.app)
```

Comment:
to my experience, not a critical thing though:
1. sometimes we might need to show loader not by redux state, but by local state(not everything should go to Redux! right?). So passing "isFetching" as a prop(probably, with better name `show` or `isShown`) is way more flexible
2. personal opinion: I prefer conditionals `{isSomethingLoading && <Loader />` over passing as a prop `<Loader isShown={isSomethingLoading} />`. With last option we can do for example:

```
{isSomethingLoading && (
  <table>{/*I don't know what exactly, maybe list  of pending requests?*/}</table>
  <Loader />
  <Button>Cancel</Button>
)}
```
when if we need to use a prop that will need be more verbose: 

```
{isSomethingLoading && (
  <table>{/*I don't know what exactly, maybe list  of pending requests?*/}</table>
  <Loader isShown={true} />
  <Button>Cancel</Button>
)}
```

See that `isShown={true}`? Well, this rarely happens to loader indicators, but I see that all the way to modals, so I personally prefer first version when no `isShown` prop at all and if parent renders `<Loader />` it means it will be shown